src/component/Mycv.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styles from '../index.css';
+import { Form,Col,ItemLayout, Icon, Input, Button, Checkbox,Select} from 'antd';
+import style from 'antd/dist/antd.css';
```

Comment:
there is no need to import this every where,  it can be import once in index or app file
see documentation - https://ant.design/docs/react/introduce